framework/doc/content/source/materials/ReporterOffsetFunctionMaterial.md

```diff
@@ -0,0 +1,24 @@
+# ReporterOffsetFunctionMaterial
```

Comment:
```suggestion
# ReporterOffsetFunctionSumMaterial
```
I had missed they perform a sum of function

I would move these to the optimization module as well. I dont see who else would use this it's very specific

Reply:
yeah let's move it all to the optimization module. Should be a couple git mv and remember to re-register from MooseApp to OptimizationApp

Reply:
I can move it. One use case example is if you have multiple source/nuemann bc terms that are all offset. So like an array of lasers.